### DISCUSSION TITLE
Is standalone jarowinkler still maintained?

### DISCUSSION THREAD

**Discussion by mgorny** [rapidfuzz/RapidFuzz#275] [General]
We've noticed that jarowinkler(-cpp) has been integrated into rapidfuzz(-cpp). Does that mean that the standalone version will effectively no longer be maintained? i'm asking as a package maintainer, primarily in context of whether keeping separate jarowinkler packages in Gentoo makes sense or whether I should remove them in favor of rapidfuzz.

**Comment by maxbachmann** [comment]
Originally `jarowinkler-cpp` and `jarowinkler` were placed in separate packages, since they use a novel algorithm. However this proved to cause a lot of unnecessary code duplication when maintaining the package. The `jarowinkler-cpp` project will not be maintained in the future. The `jarowinkler` project will probably be converted to a pure Python package which simply used the functionality which is now placed in rapidfuzz. I will continue to release the package on `PyPI` and `conda-forge`, since it isn't much work.

Since the package is not used much (pretty much always just used as dependency of rapidfuzz) I think there is little reason to keep the package in the Gentoo package index.

**Reply by mgorny** [reply]
Ok, this is what I thought. Thanks!
